tutorial/02_create-app.md

```diff
@@ -49,15 +49,15 @@ In this exercise, you will create a new Azure AD web application registration us
 
     ![Screenshot selecting the Microsoft Graph service](./images/aad-portal-newapp-graphscope.png)
 
-    Select **Application Permission**, expand the **User** group and select **User.ReadWrite.All** scope.
+    Select **Application Permission**, expand the **User** group and select **User.Read.All** scope.
 
     Select **Add permissions** to save your changes.
 
     ![Screenshot selecting Microsoft Graph scope](./images/aad-portal-newapp-graphscope-02.png)
```

Comment:
@jasonjoh , if you agree with the change i proposed, can you also update the screenshots to reflect it correctly?